packages/angular/cli/BUILD.bazel

```diff
@@ -312,19 +307,22 @@ jasmine_node_test(
     srcs = [":angular-cli_test_lib"],
 )
 
-# @external_begin
```

Comment:
I think you'll need to keep the
```
# @external_begin
```
and
```
# @external_end
```
around the genrule & pkg_npm so that these are stripped when being synced into the google3 repo; might be fine but the targets themselves are not needed in google internal so better to leave them out

Reply:
Nevermind. Not needed as per https://github.com/angular/angular-cli/pull/22258#discussion_r759582051